.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,4 @@
+contact_links:
+  - name: Muse Hub issue
+    url: https://musehub.zendesk.com/
+    about: Issues with the Muse Hub application, like installation issues
```

Comment:
Not sure what would be the best here; maybe "Issues with the Muse Hub application, rather than MuseScore itself" is better

Reply:
Perhaps this would be clearer:

> **Muse Hub issues**
> All issues to do with installing applications and sounds via Muse Hub.

I think it should be clear that includes other Muse Hub problems, like usability issues, even if they are are not directly related to installing apps and sounds. I'm more worried about non-Hub problems being reported on the Muse Hub tracker (e.g. installation issues not relating to Muse Hub, or issues using Muse Sounds in MuseScore).

Reply:
Yeah, changed it to your suggestion!